src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -32,23 +33,29 @@ export const AuxiliaryDropdown = () => {
       to: routes.hosts,
       onClick: () => sendEvent({ name: "Click All Hosts Link" }),
     },
+    {
+      text: "Task Queue",
+      to: getTaskQueueRoute(""),
+      onClick: () => sendEvent({ name: "Click Task Queue Link" }),
+    },
     {
       "data-cy": "legacy_route",
       href: `${uiURL}${legacyRoutes.distros}`,
       text: "Distros",
       onClick: () => sendEvent({ name: "Click Distros Link" }),
     },
-    {
-      text: "Project Settings",
-      to: getProjectSettingsRoute(mostRecentProject),
-      onClick: () => sendEvent({ name: "Click Projects Link" }),
-    },
+
     {
       "data-cy": "auxiliary-dropdown-project-patches",
       to: getProjectPatchesRoute(mostRecentProject),
       text: "Project Patches",
       onClick: () => sendEvent({ name: "Click Project Patches Link" }),
     },
+    {
+      text: "Project Settings",
+      to: getProjectSettingsRoute(mostRecentProject),
+      onClick: () => sendEvent({ name: "Click Projects Link" }),
```

Comment:
This may be an opportunity to change this analytics event to say `"Click Project Settings Link"`, though I don't know if that would mess up our analytics